src/Data/Vec/Functional/Relation/Binary/Permutation/Properties.agda

```diff
@@ -0,0 +1,44 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of permutation
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Vec.Functional.Relation.Binary.Permutation.Properties where
+
+open import Level using (Level)
+open import Data.Product using (_,_; proj₁; proj₂)
+open import Data.Nat using (ℕ)
+open import Data.Fin.Permutation using (id; flip; _⟨$⟩ʳ_; inverseʳ; _∘ₚ_)
+open import Data.Vec.Functional
+open import Data.Vec.Functional.Relation.Binary.Permutation
+open import Relation.Binary.PropositionalEquality
+  using (refl; trans; _≡_; cong; module ≡-Reasoning)
+
+open ≡-Reasoning
+
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+    n : ℕ
+    xs ys zs : Vector A n
+
+↭-refl : xs ↭ xs
```

Comment:
Could you use the definitions in `Relation.Binary.Indexed.Heterogeneous.Definitions` in this type and the types of the other definitions in this module?

Reply:
I added these definitions.

Reply:
Could you change the changelog to use them too?